app/controllers/application_controller/ci_processing.rb

```diff
@@ -522,63 +520,33 @@ def live_migrate_form_fields
 
   def live_migrate_vm
     assert_privileges("instance_live_migrate")
-    @record = find_record_with_rbac(VmOrTemplate, params[:id])
     case params[:button]
     when "cancel"
-      add_flash(_("Live Migration of %{model} \"%{name}\" was cancelled by the user") % {
-        :model => ui_lookup(:table => "vm_cloud"), :name => @record.name
+      add_flash(_("Live migration of %{models} was cancelled by the user") % {
+        :models => ui_lookup(:tables => "vm_cloud")
```

Comment:
I know this has been there before, but for the sake of translations, we'd like to get rid of
`ui_lookup :tables => 'static_string'`. So this should be replaced with a plain:
```
"Live migration of Instances was cancelled by the user"
```